app/src/main/java/com/example/whereto/fragments/PlanTripFragment.java

```diff
@@ -1,21 +1,228 @@
+        questionList.add(new Question("arts & entertainment"));
+        questionList.add(new Question("outlet stores/shopping centres/souvenir shops"));
+        questionList.add(new Question("nightlife/bars"));
+        questionList.add(new Question("beauty & spas"));
+
+        final QuestionsAdapter adapter = new QuestionsAdapter(questionList, view.getContext());
+
+        cardStack.setAdapter(adapter);
+
+        cardStack.setEventCallback(new SwipeDeck.SwipeEventCallback() {
+            @Override
+            public void cardSwipedLeft(int position) {}
+
+            @Override
+            public void cardSwipedRight(int position) {
+                String prompt = (questionList.get(position)).getPrompt();
+
+                switch (prompt) {
+                    case "food/restaurants":
+                        food = true;
+                        break;
+                    case "hotels":
+                        hotels = true;
+                        break;
+                    case "tours":
+                        tours = true;
+                        break;
+                    case "active/athletic activities":
+                        athletic = true;
+                        break;
+                    case "arts & entertainment":
+                        arts = true;
+                        break;
+                    case "outlet stores/shopping centres/souvenir shops":
+                        shopping = true;
+                        break;
+                    case "nightlife/bars":
+                        nightlife = true;
+                        break;
+                    default:
+                        beauty = true;
+                        break;
+                }
+            }
+
+            @Override
+            public void cardClicked(int position) {
+                Toast.makeText(view.getContext(), "Please swipe left or right", Toast.LENGTH_SHORT).show();
+            }
+
+            @Override
+            public void cardsDepleted() {}
+
+            public void cardActionDown() {
+                Toast.makeText(view.getContext(), "Please do not swipe down", Toast.LENGTH_SHORT).show();
+            }
+
+            public void cardActionUp() {
+                Toast.makeText(view.getContext(), "Please do not swipe up", Toast.LENGTH_SHORT).show();
+            }
+        });
+    }
+
+    public void getTripStartDate() {
+
+        startDate = (EditText) getView().findViewById(R.id.startDate);
+        startDate.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                final Calendar c = Calendar.getInstance();
+                int mYear = c.get(Calendar.YEAR);
+                int mMonth = c.get(Calendar.MONTH);
+                int mDay = c.get(Calendar.DAY_OF_MONTH);
+                datePickerDialog = new DatePickerDialog(v.getContext(), new DatePickerDialog.OnDateSetListener() {
```

Comment:
Need some error checking. We spoke about not having the start date after the end date. Also should check the don't start the trip in the past.